src/components/Template/Template.mdx

```diff
@@ -93,21 +87,25 @@ import {
 
 ## Accessibility
 
-Wide lines of text are difficult to read and are proven to make it harder for
-people to focus, while short lines of text force the eye to travel back too
-often, breaking a reader's rhythm. The `TemplateMain` component has a max width
-of 1280px, which is too wide for text. If your content includes long chunks of
-text, you should use the `TemplateMainNarrow` component instead. Note that
-`TemplateMainNarrow` is not meant to be used in conjunction with a sidebar.
+### Content Width
 
-This means you may use `TemplateMain` or `TemplateMainNarrow` depending on your
-needs, but the two should not be used in conjunction.
+Wide lines of text with more than 80 characters per line are difficult to read
+and are proven to make it harder for people to focus, while short lines of text
+with fewer than 30 characters per line force the eye to travel back too often,
+breaking a reader's rhythm.
```

Comment:
My vote would be to remove this paragraph and add a `Resources` link instead that talks about this. The second paragraph stands well on its own and contains the most important info.